src/Command/InstallChromeDriverCommand.php

```diff
@@ -0,0 +1,400 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Panther\Command;
+
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Throwable;
+use UnexpectedValueException;
+use ZipArchive;
+use function fclose;
+use function fopen;
+use function fwrite;
+use function preg_last_error_msg;
+use function preg_match;
+use function reset;
+use function sprintf;
+use const PHP_OS_FAMILY;
+
+final class InstallChromeDriverCommand extends Command
+{
+    public const SUCCESS = 0;
+    public const FAILURE = 1;
+
+    private const DRIVER_VERSION = 'driver-version';
+    private const CHROME_BINARY = 'chrome-binary';
+    private const DEFAULT_CHROME_BINARY = 'google-chrome';
+    private const LATEST = 'latest';
+    private const AUTO = 'auto';
+    private const OS_FAMILY = 'os-family';
+    private const WINDOWS = 'Windows';
+    private const BSD = 'BSD';
+    private const DARWIN = 'Darwin';
+    private const SOLARIS = 'Solaris';
+    private const LINUX = 'Linux';
+    private const CHROMEDRIVER_API_URL = 'https://chromedriver.storage.googleapis.com';
+    private const CHROMEDRIVER_API_VERSION_ENDPOINT = self::CHROMEDRIVER_API_URL . '/LATEST_RELEASE';
+    private const BIN_DIRECTORY = __DIR__ . '/../../bin';
+    private const DOWNLOAD_FILE_LOCATION = self::BIN_DIRECTORY . '/chromedriver.zip';
+    private const CHROMEDRIVER_BINARY_LINUX = 'chromedriver_linux64';
+    private const CHROMEDRIVER_BINARY_MAC = 'chromedriver_mac64';
+    private const CHROMEDRIVER_BINARY_WINDOWS = 'chromedriver_win32';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var ZipArchive
+     */
+    private $zip;
+
+    public function __construct(string $name, HttpClientInterface $httpClient, Filesystem $filesystem, ZipArchive $zip)
```

Comment:
These deps could be nullable and initialized directly in the constructor if not passed as parameters, so won't have to handle the DI manually in the caller script.